### PR TITLE
Upgrade Helm Version

### PR DESCRIPTION
**Pull request by NicolasT**
Fixes: GH-707
See: https://github.com/scality/metalk8s/issues/707
Fixes: GH-709
See: https://github.com/scality/metalk8s/pull/709

### CODE REVIEW

**Comment by NicolasT** [comment]
@bert-e approve

File: eve/images/pep8-checker/Dockerfile
Comment (Zempashi) [inline eve/images/pep8-checker/Dockerfile:17]: Could we fix the version here ? == 1.8.0 seems a good version here (sorry my bad, to not have the correct version pinned here), then checking if py-twisted work or if gcc and musl is needed (and in this case remove py-twisted)

Comment (NicolasT) [reply eve/images/pep8-checker/Dockerfile:17]: Done

**Comment by NicolasT** [comment]
@Zempashi I tried using the package-provided Twisted, but failed (still insisted on pulling a newer version), so had to keep the `musl-dev` etc. in.

**Comment by NicolasT** [comment]
@bert-e approve

**Review by Zempashi — APPROVED** [review]

**Comment by tcarmet** [comment]
@NicolasT Bert-e won't merge this change as we recreated the git tree. You can merge this manually (as you have admin rights)

**Comment by NicolasT** [comment]
> @NicolasT Bert-e won't merge this change as we recreated the git tree. You can merge this manually (as you have admin rights)

What about 1.2/1.3?

**Comment by tcarmet** [comment]
Right... forward port it manually?

**Comment by tcarmet** [comment]
cc @DavidPineauScality

**Comment by NicolasT** [comment]
I'll do it manually to unblock @ssalaues.

**Comment by ghost** [comment]
we have an open issue in our tooling to allow configuring the waterfall in a more fine-grained fashion, so that the forward-porting waterfall would be barred at the major version switch for 2.0
Not scheduled at the moment, though.

**Pull request merged**
